client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml

```diff
@@ -101,6 +101,16 @@
    <translation id="privateDatabase_configuration_interactive_timeout" qtlid="370200">Le nombre de secondes que le serveur attend avant de fermer la connexion lorsqu'il n'y a pas d'activité lors d'une connexion en invite de commande.</translation>
    <translation id="privateDatabase_configuration_wait_timeout" qtlid="370213">Le nombre de secondes que le serveur attend avant de fermer la connexion lorsqu'il n'y a pas d'activité lors d'une connexion externe.</translation>
 
+   <translation id="privateDatabase_configuration_field_autocommit">Définit si les requêtes sont automatiquement commitées ou non.</translation>
```

Comment:
"commitées" -> "actives" / "prise en compte" ?

"Comitées" is not really a french word ^^

Reply:
Like I said previously: 
```This is a page for the experts, may be some french canadiens would like soumettre better but commiter is more meaningful. I think thine is fine.```
I could also add that the verb ```commiter``` is already used in this traduction ```privateDatabase_configuration_autoCommit```: ```...donc si les requêtes sont automatiquement commitées ...```

Reply:
Please ask to the content team :)